### ISSUE TITLE
Исчезает табличка с никнеймом игрока при включенном голосовом чате, когда заходишь и выходишь с главного меню

### ISSUE THREAD

**Issue by Vladislav4KZ** [FWGS/xash3d-fwgs#993]
Звук с голосового чата продолжает работать.

**Comment by Vladislav4KZ** [comment]
Со второго раза баг не воспроизводится.

**Comment by Vladislav4KZ** [comment]
Со второго раза табличка не исчезала какое-то время, затем сама исчезла когда я не выключал голосовой чат.

**Comment by Vladislav4KZ** [comment]
Оказывается, открытие главного меню тут не причём. Табличка исчезает сама по себе через время.

**Comment by Vladislav4KZ** [comment]
Баг воспроизводится только в оригинальном Half-Life, в модах не воспроизводится.

**Comment by a1batross** [comment]
Появился после https://github.com/FWGS/xash3d-fwgs/commit/3c682507e7764b73864006dd386ca854d06c783f ?

**Comment by Vladislav4KZ** [comment]
В модах эта табличка исчезает по окончании проигрывании музыки в голосовом чате.

Проверял на Sven Coop 4.5

**Comment by Vladislav4KZ** [comment]
> Появился после https://github.com/FWGS/xash3d-fwgs/commit/3c682507e7764b73864006dd386ca854d06c783f ?

Да, раньше такого не замечал. Даже когда проигрывал voice_input.wav в голосовом чате.

**Comment by Vladislav4KZ** [comment]
> В модах эта табличка исчезает по окончанию проигрывании музыки в голосовом чате.
> 
> Проверял на Sven Coop 4.5

В оригинальном Half-Life теперь тоже такое поведение. Исчезает после окончания проигрывания музыки.

**Comment by a1batross** [comment]
Потому что голдсорс обновляет статус когда канал закончился, у нас по таймауту в 200мс.

Это баг, но требует больших затрат, чем пара строчек. :)
